TeminiFalegan.md

```diff
@@ -0,0 +1,43 @@
+Audience Analysis in Technical Writing 
+# Name: Temini Falegan 
+## Topic Chosen: Audience Analysis in Technical Writing
+
+### Introduction
+
+Have you ever encountered an article on a trusted website that left you confused? The language might seem overly technical, or the concepts poorly explained. This frustration often stems from a lack of **audience analysis**. 
```

Comment:
Awseome work so far! I'll like to still see other use of markdown in your article beyond bold and heading.

Reply:
Thank you! Alright, I'll add somethings now.

Reply:
I have included an image, added a hyperlink, and put in some unordered lists as well as one ordered list.